package.json

```diff
@@ -11,8 +11,20 @@
     "packages/*"
   ],
   "devDependencies": {
+    "all-contributors-cli": "^6.8.1",
     "lerna": "^3.17.0",
-    "react": "16.9.0",
-    "all-contributors-cli": "^6.8.1"
+    "lerna-changelog": "^0.8.2",
+    "react": "16.9.0"
+  },
+  "changelog": {
```

Comment:
Please move the changelog configs to [lerna.json](https://github.com/fidisys/react-native-components/blob/master/lerna.json) file, in that way we can keep all the lerna related things in once place 😄